lib/Controller/RoomController.php

```diff
@@ -45,901 +47,958 @@
 use OCP\IGroup;
 use OCP\IGroupManager;
 
-class RoomController extends OCSController {
```

Comment:
Something is wrong here - the diff contains the whole class. Did you change the indentation?